composer/algorithms/gated_linear_units/gated_linear_unit_layers.py

```diff
@@ -0,0 +1,65 @@
+# Copyright 2022 MosaicML Composer authors
+# SPDX-License-Identifier: Apache-2.0
+
+from typing import Callable
+
+import torch
+
+
+class DummyBERTIntermediateOutput(torch.nn.Module):
```

Comment:
Maybe a bit more descriptive..e.g. `IdentityLayer`, since its not specific to BERT really.